flows/actions/testdata/call_classifier.json

```diff
@@ -60,7 +55,18 @@
                 }
             ],
             "waiting_exits": [],
-            "parent_refs": []
+            "parent_refs": [],
+            "problems": [
+                {
+                    "type": "missing_dependency",
+                    "node_uuid": "72a1f5df-49f9-45df-94c9-d86f7ea064e5",
+                    "action_uuid": "ad154980-7bf7-4ab8-8728-545fd6378912",
+                    "reference": {
+                        "uuid": "63998ee7-a7a5-4cc5-be67-c773e1b6b9b1",
```

Comment:
Don't we need the type of the reference somewhere?

Reply:
Added now